content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: '.getActualMinimum()'
+Description: 'Returns the minimum value that the specified calendar field could have based on the time value of this calendar'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Calendar'
+  - 'Time'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The `.getActualMinimum()` method returns the minimum possible value that the specified calendar field could have 
+
+## Syntax
+```pseudo
+//Instance of a calendar is created
+Calendar cal = Calendar.getInstance();
+
+//This returns the minimum value of the calendar field provided
+min = cal.getActualMinimum(Calendar.YEAR);
+```
+
+The method above returns the minimum value for a calendar year
+
+## Example
+
+```java
```

Comment:
Before the example should be an introductory sentence summarizing what the example is doing.